Add note that cross-device flow diagram doesn't show all parameters (#625)
openid/OpenID4VP: openid-4-verifiable-presentations-1_0.md

In this flow, the Verifier prepares an Authorization Request and renders it as a QR Code. The End-User then uses the Wallet to scan the QR Code. The Presentations are sent to the Verifier in a direct HTTP POST request to a URL controlled by the Verifier. The flow uses the Response Type `vp_token` in conjunction with the Response Mode `direct_post`, both defined in this specification. In order to keep the size of the QR Code small and be able to sign and optionally encrypt the Request Object, the actual Authorization Request contains just a Request URI according to [@!RFC9101], which the wallet uses to retrieve the actual Authorization Request data.

> Note: The diagram does not illustrate all parameters and optional features of this specification.

Reviewer: ```suggestion
Note: The diagram does not illustrate all parameters and all optional features of this specification.
```

Author: What I wrote was definitely not good English 😂  maybe this?

```suggestion
Note: The diagram illustrates neither all parameters nor all optional features of this specification.
```
